cvat/apps/events/views.py

```diff
@@ -0,0 +1,151 @@
+                description="ID of query request that need to check or download"),
+        ],
+        responses={
+            '200': OpenApiResponse(description='Download of file started'),
+            '201': OpenApiResponse(description='CSV log file is ready for downloading'),
+            '202': OpenApiResponse(description='Creating a CSV log file has been started'),
+        })
+    @action(detail=False, methods=['GET', 'POST'])
+    def events(request):
+        if request.method == 'POST':
+            serializer = ClientEventsSerializer(data=request.data)
+            serializer.is_valid(raise_exception=True)
+
+            org = request.iam_context['organization']
+            org_id = org.id if org else None
+
+            send_time = datetime_parser.isoparse(serializer.data["send_timestamp"])
+            receive_time = datetime.datetime.now(datetime.timezone.utc)
+            time_correction = receive_time - send_time
+            last_timestamp = None
+
+            for event in serializer.data["events"]:
+                event['user'] = request.user.id
+                timestamp = datetime_parser.isoparse(event['timestamp'])
+                if last_timestamp:
+                    t_diff = timestamp - last_timestamp
+                    if t_diff.seconds < EventsViewSet.TIME_THRESHOLD:
+                        payload = event.get('payload', {})
+                        if payload:
+                            payload = json.loads(payload)
+
+                        payload['working_time'] = t_diff.microseconds // 1000
+                        event['payload'] = json.dumps(payload)
+
+                last_timestamp = timestamp
+                event['timestamp'] = str((timestamp + time_correction).timestamp())
+                event['source'] = 'client'
+                event['organization'] = org_id
+                message = JSONRenderer().render(event).decode('UTF-8')
+                jid = event.get("job")
+                tid = event.get("task")
+                if jid:
+                    clogger.job[jid].info(message)
+                elif tid:
+                    clogger.task[tid].info(message)
+                else:
+                    clogger.glob.info(message)
+
+                vlogger.info(message)
+
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        elif request.method == 'GET':
+            return export(
+                request=request,
+                queue_name=settings.CVAT_QUEUES.EXPORT_DATA.value,
+            )
+
+    @staticmethod
+    @extend_schema(summary='Method saves an exception from a client on the server',
+        description='Sends logs to the ELK if it is connected',
+        request=EventSerializer, responses={
+            '201': ClientEventsSerializer,
+        })
+    @action(detail=False, methods=['POST'], serializer_class=EventSerializer)
+    def exception(request):
```

Comment:
Do we really want to have a separate endpoint for exceptions? From my perspective is a regular event.